docs/03_server/05_access-control/03_password_authentication.md

```diff
@@ -325,29 +325,63 @@ All requests below are capable of returning an error with a code of INTERNAL_ERR
 ---
 
 ## Pre-authentication
-Pre-authentication messages can be sent by a client without the user being logged in.
+Pre-authentication messages can be sent by a client without the user being logged in. There are four messages that can be used without being logged in.
 
-### Login preferences
-You must make sure that any connecting client knows the types of functionality that you have configured on the security module. For example, you could offer the client two ways of resetting user passwords: either via an administrator or by sending an email.  This choice can affect how the login dialog is displayed, so it is vital that the connecting client knows this before any user logs in.
-Currently, this is the only preference published.
-#### Request
-    MESSAGE_TYPE = EVENT_LOGIN_PREFS
-#### Response
-    MESSAGE_TYPE = EVENT_LOGIN_PREFS_ACK
-        DETAILS.PASSWORD_RESET_TYPE = ADMIN/EMAIL
+### Self-service password reset
```

Comment:
I am describing 3 different message flows that can happen before being authenticated here, as well as removing EVENT_LOGIN_PREFS given it doesn't exist.